Extension/bin/messages/cs/messages.json

```diff
@@ -0,0 +1,3194 @@
+[
+  "Neznámá chyba",
+  "poslední řádek souboru končí bez nového řádku",
+  "poslední řádek souboru končí zpětným lomítkem",
+  "Soubor #include %sq obsahuje sám sebe.",
+  "nedostatek paměti",
+  null,
```

Comment:
Why is it `null` here?  What happens in this case?

Reply:
nevermind, I think those messages have been removed but the ids stayed the same.